doc/changelog.rst

```diff
@@ -3,6 +3,72 @@
 Changelog
 =========
 
+v2.9.0
+------
+Highlights
+
+TODO: add highlights
```

Comment:
@ESMValGroup/esmvaltool-coreteam If you have suggestions for some highlights for this release, please comment here.

Reply:
Dask distributed and Python 311 :beer:

Reply:
Python 3.11 was already the highlight of [last week's release](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/changelog.html#v2-8-1) ;-)